src/ZipStream.php

```diff
@@ -325,8 +325,9 @@ public function addFileFromPsr7Stream(
     public function finish(): void
     {
         // add trailing cdr file records
-        foreach ($this->files as $file) {
-            $file->addCdrFile();
+        foreach ($this->files as $data) {
```

Comment:
Can you also remove "data" here please? foreach files as file conveys better the intent.